docker-scripts/env-command.sh

```diff
@@ -8,6 +8,52 @@ echo "--------------------------------------------------------------------------
 
 export PS1='\[\e]0;\w\a\]\n[\#] \[\e[32m\u@Cloud Pak Deployer:\[\e[33m\]\w \e[m\$ ';
 
+# 0 - not exist; 1 - exists
+ocp_ssh_private_key_exists=$($SCRIPT_DIR/../cp-deploy.sh vault list |grep "ocp-ssh-private-key" |wc -l)
+ocp_ssh_pub_key_exists=$($SCRIPT_DIR/../cp-deploy.sh vault list |grep "ocp-ssh-pub-key" |wc -l)
+
+#Create the ssh key pair
+if [ "$ocp_ssh_private_key_exists" == 1 ] && [ "$ocp_ssh_pub_key_exists" == 1 ]; then
+
+  ocp_ssh_private_result=$($SCRIPT_DIR/../cp-deploy.sh vault get -e ANSIBLE_STANDARD_OUTPUT=false -vs "ocp-ssh-private-key")
```

Comment:
This is a bit of an awkward way to set the value in the file. ./cp-deploy.sh vault get -vs <secret> -vsf <secret_file> will get the content of the secret directly into the secret_file, retaining any new-line characters.